### ISSUE TITLE
Check that all episode objectives are assessed

### ISSUE THREAD

**Issue by svenvanderburg** [carpentries-incubator/deep-learning-intro#272]
Check that all lesson and episode objectives are assessed by exercises or another opportunity for formative assessment.

**Comment by svenvanderburg** [comment]
See also https://github.com/carpentries-incubator/deep-learning-intro/issues/264#issuecomment-1332224716

**Comment by svenvanderburg** [comment]
### Introduction:
- "Recall the sort of problems for which Deep Learning is a useful tool"
- "List some of the available tools for Deep Learning"
- "Recall the steps of a Deep Learning workflow"
- "Explain why it is important to test the accuracy of a Deep Learning model."
- "Identify the inputs and outputs of a deep neural network."
- "Test that you have successfully installed Deep Learning libraries"

### 02-keras
- "Use the deep learning workflow to structure the notebook"
- "Explore the dataset using pandas and seaborn"
- "Use one-hot encoding to prepare data for classification in Keras"
- "Describe a fully connected layer"
- "Implement a fully connected layer with Keras"
- "Use Keras to train a small fully connected network on prepared data"
- "Plot the loss curve of the training process"
- "Use a confusion matrix to measure the trained networks' performance on a test set"

### 03-monitor-the-model
- "Explain the importance of splitting the data in a train and test set"
- "Use the data splits to plot the training process"
- "Set the training goal for your deep neural network"
- "Measure the performance of your deep neural network"
- "Interpret the training plots to recognize overfitting"
- "Implement basic strategies to prevent overfitting"

### 04-advanced-layer-types
- "Understand why convolutional and pooling layers are useful for image data"
- "Use normalization as preparation step for Deep Learning"
- "Implement a convolutional neural network on an image dataset"
- "Plot and interpret the training process"
- "Do a visual inspection of the results of your neural network"
- "Understand strategies to improve your model based on the plots"
- "Use a drop-out layer to prevent overfitting"
